app/controllers/answers_controller.rb

```diff
@@ -66,4 +77,22 @@ def set_answer
     def answer_params
       params.require(:answer).permit(:answer)
     end
+
+    
+
+    def initialize_search
+      @answer = Answer.order(answer: :desc)
```

Comment:
You can set the default order in the Answer model.
You probably want to name the instance variable as `@answers`